tests/utils/compute_groundtruth.cpp

```diff
@@ -353,6 +356,28 @@ int aux_main(const std::string &base_file, const std::string &query_file,
   int num_parts = get_num_parts<T>(base_file.c_str());
   load_bin_as_float<T>(query_file.c_str(), query_data, nqueries, dim, 0);
 
+  // load tags
+  std::unordered_map<std::uint32_t, std::uint32_t> location_to_tag;
+  if (include_tags) {
+      size_t file_dim, file_num_points;
```

Comment:
Could you please run clang-format to normalize the indentation to the rest of the repo